src/asim/configs/resident/summarize_preprocessor.csv

```diff
@@ -0,0 +1,434 @@
+,_knrout_prm_wlk_totivt,trip_odt_skims['KNROUT_PRM_TOTALIVTT']
+,_pnrout_loc_wlk_totivt,trip_odt_skims['PNROUT_LOC_TOTALIVTT']
+,_pnrout_mix_wlk_totivt,trip_odt_skims['PNROUT_MIX_TOTALIVTT']
+,_pnrout_prm_wlk_totivt,trip_odt_skims['PNROUT_PRM_TOTALIVTT']
+,_tncout_loc_wlk_totivt,trip_odt_skims['TNCOUT_LOC_TOTALIVTT']
+,_tncout_mix_wlk_totivt,trip_odt_skims['TNCOUT_MIX_TOTALIVTT']
+,_tncout_prm_wlk_totivt,trip_odt_skims['TNCOUT_PRM_TOTALIVTT']
+,_walk_loc_wlk_totivt,trip_odt_skims['WALK_LOC_TOTALIVTT']
+,_walk_mix_wlk_totivt,trip_odt_skims['WALK_MIX_TOTALIVTT']
+,_walk_prm_wlk_totivt,trip_odt_skims['WALK_PRM_TOTALIVTT']
+,_knrin_loc_wlk_dwell,trip_odt_skims['KNRIN_LOC_DWELLTIME']
+,_pnrin_loc_wlk_dwell,trip_odt_skims['PNRIN_LOC_DWELLTIME']
+,_tncin_loc_wlk_dwell,trip_odt_skims['TNCIN_LOC_DWELLTIME']
+,_knrout_loc_wlk_dwell,trip_odt_skims['KNROUT_LOC_DWELLTIME']
+,_pnrout_loc_wlk_dwell,trip_odt_skims['PNROUT_LOC_DWELLTIME']
+,_tncout_loc_wlk_dwell,trip_odt_skims['TNCOUT_LOC_DWELLTIME']
+,_walk_loc_wlk_dwell,trip_odt_skims['WALK_LOC_DWELLTIME']
+
+
+# Initial wait time (FIRSTWAIT)
+,_knrin_loc_wlk_iwait,trip_odt_skims['KNRIN_LOC_FIRSTWAIT']
+,_knrin_mix_wlk_iwait,trip_odt_skims['KNRIN_MIX_FIRSTWAIT']
+,_knrin_prm_wlk_iwait,trip_odt_skims['KNRIN_PRM_FIRSTWAIT']
+,_pnrin_loc_wlk_iwait,trip_odt_skims['PNRIN_LOC_FIRSTWAIT']
+,_pnrin_mix_wlk_iwait,trip_odt_skims['PNRIN_MIX_FIRSTWAIT']
+,_pnrin_prm_wlk_iwait,trip_odt_skims['PNRIN_PRM_FIRSTWAIT']
+,_tncin_loc_wlk_iwait,trip_odt_skims['TNCIN_LOC_FIRSTWAIT']
+,_tncin_mix_wlk_iwait,trip_odt_skims['TNCIN_MIX_FIRSTWAIT']
+,_tncin_prm_wlk_iwait,trip_odt_skims['TNCIN_PRM_FIRSTWAIT']
+,_knrout_loc_wlk_iwait,trip_odt_skims['KNROUT_LOC_FIRSTWAIT']
+,_knrout_mix_wlk_iwait,trip_odt_skims['KNROUT_MIX_FIRSTWAIT']
+,_knrout_prm_wlk_iwait,trip_odt_skims['KNROUT_PRM_FIRSTWAIT']
+,_pnrout_loc_wlk_iwait,trip_odt_skims['PNROUT_LOC_FIRSTWAIT']
+,_pnrout_mix_wlk_iwait,trip_odt_skims['PNROUT_MIX_FIRSTWAIT']
+,_pnrout_prm_wlk_iwait,trip_odt_skims['PNROUT_PRM_FIRSTWAIT']
+,_tncout_loc_wlk_iwait,trip_odt_skims['TNCOUT_LOC_FIRSTWAIT']
+,_tncout_mix_wlk_iwait,trip_odt_skims['TNCOUT_MIX_FIRSTWAIT']
+,_tncout_prm_wlk_iwait,trip_odt_skims['TNCOUT_PRM_FIRSTWAIT']
+,_walk_loc_wlk_iwait,trip_odt_skims['WALK_LOC_FIRSTWAIT']
+,_walk_mix_wlk_iwait,trip_odt_skims['WALK_MIX_FIRSTWAIT']
+,_walk_prm_wlk_iwait,trip_odt_skims['WALK_PRM_FIRSTWAIT']
+
+# Transfer wait time (XWAIT)
+,_knrin_loc_wlk_xwait,trip_odt_skims['KNRIN_LOC_XFERWAIT']
+,_knrin_mix_wlk_xwait,trip_odt_skims['KNRIN_MIX_XFERWAIT']
+,_knrin_prm_wlk_xwait,trip_odt_skims['KNRIN_PRM_XFERWAIT']
+,_pnrin_loc_wlk_xwait,trip_odt_skims['PNRIN_LOC_XFERWAIT']
+,_pnrin_mix_wlk_xwait,trip_odt_skims['PNRIN_MIX_XFERWAIT']
+,_pnrin_prm_wlk_xwait,trip_odt_skims['PNRIN_PRM_XFERWAIT']
+,_tncin_loc_wlk_xwait,trip_odt_skims['TNCIN_LOC_XFERWAIT']
+,_tncin_mix_wlk_xwait,trip_odt_skims['TNCIN_MIX_XFERWAIT']
+,_tncin_prm_wlk_xwait,trip_odt_skims['TNCIN_PRM_XFERWAIT']
```

Comment:
There is no 'TNCIN_LOC_XFERWAIT', 'TNCIN_MIX_XFERWAIT', 'TNCIN_PRM_XFERWAIT' in transit skims